packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx

```diff
@@ -272,9 +272,36 @@ export const OAuthLoginPage = () => {
 
   return (
     <ContentWrapper>
-      <div className={cn(styles.centeredContent, styles.titleContainer)}>
-        <h1 className={styles.title}>{titleText}</h1>
-      </div>
+      <Flex alignItems='center' direction='column'>
+        <Flex gap='l' alignItems='center' mb='l'>
+          <Flex h='88px' w='88px'>
+            <img src={AppIcon} alt={'Audius Logo'} />
+          </Flex>
+          <IconTransaction color='default' />
+          <Flex h='88px' w='88px'>
+            {appImage ? (
+              <img src={appImage} alt={`${appName} Image`} />
+            ) : (
+              <Flex
+                w='100%'
+                justifyContent='center'
+                alignItems='center'
+                borderRadius='l'
+                css={{ backgroundColor: 'var(--harmony-n-200)' }}
+              >
+                <IconEmbed
+                  color='subdued'
+                  css={{ width: '48px', height: '48px' }}
```

Comment:
I'm sure there's probably a harmony constant that should be used here. But I'm not certain what it is 😅